third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,82 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"Deny", "Deny": {"Information": "Access is denied"}}, "policy": "Deny by default"}
+}
+
+verdict[outputFormatted] {
+	count(rule) > 0
+	output.action.name == "Deny"	
+	output = rule[_]
+	outputFormatted := {"action": {"name":"Deny", "Deny": {"Information": "Access is denied"}}, "policy": output.policy}
+}
+
+verdict[outputFormatted] {
+	count(rule) > 0
+	output.action.name != "Deny"
```

Comment:
Remove this line instead of having the previous verdict block

Reply:
Removed.